From dd09c14c1009467f99b2fdecdd686e4a664fce9d Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:38:35 -0400
Subject: [alsa] hda: fix a typo by changing mute_led_nid to cap_mute_led_nid

Message-id: <140972823573-384-patchset-jkysela@redhat.com>
Patchwork-id: 91080
O-Subject: [RHEL 7.1 PATCH 383/442] ALSA: hda - fix a typo by changing mute_led_nid to cap_mute_led_nid
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

BugLink: https://bugs.launchpad.net/bugs/1329580
Cc: David Henningsson <david.henningsson@canonical.com>
Cc: Kailang Yang <kailang@realtek.com>
Signed-off-by: Hui Wang <hui.wang@canonical.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: fc1fad93c95bed53f701a1169c556113341b43f0)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 327413e..73b1759 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -3529,7 +3529,7 @@ static void alc269_fixup_hp_cap_mic_mute_hook(struct hda_codec *codec,
  struct alc_spec *spec = codec->spec;
  unsigned int pinval, enable, disable;
 
- pinval = snd_hda_codec_get_pin_target(codec, spec->mute_led_nid);
+ pinval = snd_hda_codec_get_pin_target(codec, spec->cap_mute_led_nid);
  pinval &= ~AC_PINCTL_VREFEN;
  enable  = pinval | AC_PINCTL_VREF_80;
  disable = pinval | AC_PINCTL_VREF_HIZ;
-- 
1.7.1